ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -183,8 +182,26 @@ data LowLevelRunNodeArgs m addrNTN addrNTC versionDataNTN versionDataNTC blk
                          (p2p :: Diffusion.P2P) =
    LowLevelRunNodeArgs {
 
-      -- | How to manage the clean-shutdown marker on disk
-      llrnWithCheckedDB :: forall a. (LastShutDownWasClean -> m a) -> m a
+      -- | A bracket function that manages the clean-shutdown marker on disk.
+      --
+      -- - If the marker is missing on startup, then ChainDB initialization will
+      --   revalidate the database contents.
+      --
+      -- - If the OS kills the nodes, then we don't even have the opportunity to
+      --   write out the marker file, which is appropriate, since we want the
+      --   next startup to do revalidation.
```

Comment:
Is "even" necessary? Also, I don't know if "appropriate" is the right word here. I don't have a good alternative, I'd just say:

```
which is fine, since ...
```

Reply:
Done (in the other comment as suggested) and reworded this one.